.devcontainer/devcontainer.json

```diff
@@ -22,19 +20,22 @@
 	},
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "uname -a",
-
-	// Configure tool-specific properties.
-	// "customizations": {},
+	// Here we run `colcon build` to build the workspace initially, (to speed up later builds), and we keep going if it fails for some packages to get as much compiled as possible
+	"postCreateCommand": ".devcontainer/post-create.sh",
```

Comment:
this is the command that is run once the devcontainer is up. That's what we use to configure `~/.bashrc` and also attempt a colcon build, to speed up later calls of colcon. See `.devcontainer/post-create.sh`

Reply:
This is very nice! once I relanched the container with the new update it did automaticaly build the workspace